content/en/docs/tasks/traffic-management/egress/egress-control/index.md

```diff
@@ -389,9 +389,47 @@ servicesIpv4Cidr: 10.7.240.0/20
 
 Use `--set values.global.proxy.includeIPRanges="10.4.0.0/14\,10.7.240.0/20"`
 
-#### Azure Container Service(ACS)
+#### Azure Kubernetes Service (AKS)
 
-Use `--set values.global.proxy.includeIPRanges="10.244.0.0/16\,10.240.0.0/16`
+##### Kubenet
+
+To see which service CIDR and pod CIDR are used in the cluster, use `az aks show` and look for the `serviceCidr`:
+
+{{< text bash >}}
+$ az aks show --resource-group ${RESOURCE_GROUP} --name ${CLUSTER} | grep Cidr
+    "podCidr": "10.244.0.0/16",
+    "podCidrs": [
+    "serviceCidr": "10.0.0.0/16",
+    "serviceCidrs": [
+{{< /text >}}
+
+Then use `--set values.global.proxy.includeIPRanges="10.244.0.0/16\,10.0.0.0/16"`
+
+##### Azure CNI
```

Comment:
Just remembered, we should also add a caveat that the Azure CNI steps here are for non-overlay networking modes. With Azure CNI overlay, the steps should be the same as Kubenet for getting PodCIDR and ServiceCIDR.